apps/web/components/eventtype/Locations.tsx

```diff
@@ -433,7 +433,7 @@ const Locations: React.FC<LocationsProps> = ({
         <p className="text-default mt-2 text-sm">
           <Trans i18nKey="cant_find_the_right_video_app_visit_our_app_store">
```

Comment:
FYI - The text i.e - `cant_find_the_right_video_app_visit_our_app_store` still says "Can't find the right video app". So, please let me know if this has to be also changed to something else.

Reply:
Let's make it to conferencing, when everything is conferencing instead of video